skeleton/aws/README.md

```diff
@@ -139,6 +147,17 @@ The new variable `MY_NEW_VAR` will be available in the ECS task definition.
 
 **Note:** Re-deploying the application is required when updating sensitive variables.
 
+### Step 5: Adjust the ALB Target Group Health Check to your need
+
+The default settings for the Application Load Balancer Target Group health check are:
+
+- interval: every 5 seconds
+- timeout: 3 seconds
+- 2 consecutive failed checks will mark the task as failed and will trigger a rollback
+
+This suits well applications that are quickly up and running (e.g. static website, Go Gin app, ...).
+If your application load time is slower (e.g. Ruby on Rails), consider reviewing these numbers for longer timeout and interval.
```

Comment:
> consider reviewing these numbers

Can we share the actual values used by the Flexi project? This would be beneficial for the next RoR project that intends to use this template

Reply:
I still need to readjusts based on the CloudWatch logs (I'll do that this week) becasue currently I setup `60 seconds timeout and 120 seconds intervals) but it is probably too much (with 2 consecutive failed, it means a user experimenting a down instance will have to wait at lease 4 minutes before the instance will start to be removed).

Reply:
Measured on our 2 last deployments (1 yesterday and one now): From the first app load log to the "listening on 127.0.0.1:3000" message, it took 44 seconds (yesterday) and 35 seconds (today). So I think I can safely divide by 2 the current numbers to reach:
- check every 1 min
- timeout after 30 seconds

(and keep the 2 consecutive fails before marking a version as failing).

Reply:
a74d39f ✅ Let me know if you have a different idea 🙌

Reply:
Thanks! It sounds good for me